docs/src/peculiarities.md

```diff
@@ -0,0 +1,15 @@
+# Peculiarities
+
+## `_True` and `_False` (or `BoolsAsTypes`)
```

Comment:
Internals? We should remove the underscore if we document this, or use the existing Static.jl package.

But probably users can use `true` and `false` and we can keep this hidden.

Reply:
In a loop for example it may make sense for users to use `_True` instead of `true`.  But we could definitely switch to Static!

Reply:
Probably we should try to make sure constprop from true => _True just works, and if not switch to Static.jl

Reply:
maybe we are just missing som `@inline` and/or `Base.@constprop :aggressive`